_wallets/trezormodelt.md

```diff
@@ -0,0 +1,27 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: trezormodelt
+title: "Trezor Model T"
+titleshort: "Trezor Model T"
+compat: "hardware"
+level: 2
+platform:
+  - hardware:
+    name: hardware
+    os:
+      - name: hardware
+        text: "wallettrezormodelt"
+        link: "https://trezor.io/"
+        source: "https://github.com/trezor/trezor-firmware"
+        screenshot: "trezormodelt.jpg"
```

Comment:
Please include the screenshot as a png instead of a jpg.

Reply:
0194369187cca5057e4a96fdcff57cab5b632a30